stories/modules/featured-content.stories.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react'
+
+import { FeaturedContent } from '../../src/modules'
+
+export default {
+  title: 'Modules/Featured Content',
+}
+
+export const Normal = () => {
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false)
+    }, 2000)
+  })
+
+  const renderHello = () => (
+    <>
+      <div className='min-w-72 h-10 bg-red-500'>loading</div>
+    </>
+  )
+
+  const renderTest = () => (
+    <>
+      <div className='min-w-48 h-48 mr-2 bg-success-500'>Test</div>
```

Comment:
This is just a general opinion. I would choose another text instead of `Text` to add here and place it in the middle of the div. @sallkall perhaps can give an idea.

Reply:
Perhaps we can also refactor this repetition by using 
```
new Array(22).map (() => <div className='min-w-48 h-48 mr-2 bg-success-500'>Test</div>) ?
```